people.html

```diff
@@ -572,6 +529,7 @@ <h2>Past Members</h2>
 
         <h3>Postdoctoral Researchers</h3>
         <ul>
+            <li>Shaiful Chowdhury</li>

```

Comment:
Would be nice to add where they are right now (if known)

Reply:
Currently postdoc at University of Calgary.